librz/core/rop.c

```diff
@@ -224,16 +224,15 @@ static bool rz_rop_print_json_mode(const RzCore *core, const RzCoreAsmHit *hit,
 	return true;
 }
 
-RZ_IPI void rz_core_rop_reg_info_free(RzRopRegInfo *reg_info) {
-	if (!reg_info) {
-		return;
-	}
+RZ_API void rz_core_rop_reg_info_free(RZ_NULLABLE RzRopRegInfo *reg_info) {
+	rz_return_if_fail(reg_info);
```

Comment:
If it's `RZ_NULLABLE` as it should be for any `_free()` function, there should not be assert. Please restore the original NULL pointer check-and-return.

Reply:
Agree :+1: Changing this. Also can we put this up in the doc?

Reply:
Sure, please send a separate PR. Thanks for finding a weak spot in our developer documentation.